+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Feb  2 23:04:04 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL nmon
+ pkgs=nmon
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs nmon --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpcvz23xsb
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpcvz23xsb
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Feb  2 23:04:28 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Feb  2 23:04:29 2023 - INFO  - Start to run test.
+ expect
spawn nmon
+
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_+
+ CHECK_RESULT 1 0 0 'Failed option: +'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: +'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: +'
+ LOG_ERROR 'Failed option: +'
+ message='Failed option: +'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: +'
Thu Feb  2 23:04:30 2023 - ERROR - Failed option: +
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 39'
+ message='oe_test_nmon_04.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 39'
Thu Feb  2 23:04:31 2023 - ERROR - oe_test_nmon_04.sh line 39
+ return 0
+ expect
spawn nmon -s4
-
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_-
+ CHECK_RESULT 1 0 0 'Failed option: -'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: -'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: -'
+ LOG_ERROR 'Failed option: -'
+ message='Failed option: -'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: -'
Thu Feb  2 23:04:32 2023 - ERROR - Failed option: -
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 50'
+ message='oe_test_nmon_04.sh line 50'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 50'
Thu Feb  2 23:04:33 2023 - ERROR - oe_test_nmon_04.sh line 50
+ return 0
+ expect
spawn nmon
l0
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_0
+ CHECK_RESULT 1 0 0 'Failed option: 0'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 0'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 0'
+ LOG_ERROR 'Failed option: 0'
+ message='Failed option: 0'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 0'
Thu Feb  2 23:04:34 2023 - ERROR - Failed option: 0
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 61'
+ message='oe_test_nmon_04.sh line 61'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 61'
Thu Feb  2 23:04:34 2023 - ERROR - oe_test_nmon_04.sh line 61
+ return 0
+ expect
spawn nmon
1
Error opening terminal: unknown.
expect: spawn id exp3 not open
    while executing
"expect "Nice" {
        exec touch ./interactive_1_2
    }"
+ test -f ./interactive_1_1
+ CHECK_RESULT 1 0 0 'Failed option: 1'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 1'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 1'
+ LOG_ERROR 'Failed option: 1'
+ message='Failed option: 1'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 1'
Thu Feb  2 23:04:35 2023 - ERROR - Failed option: 1
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 82'
+ message='oe_test_nmon_04.sh line 82'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 82'
Thu Feb  2 23:04:36 2023 - ERROR - oe_test_nmon_04.sh line 82
+ return 0
+ expect
spawn nmon
3
Error opening terminal: unknown.
expect: spawn id exp3 not open
    while executing
"expect "CPU" {
        exec touch ./interactive_3_2
    }"
+ test -f ./interactive_3_1
+ CHECK_RESULT 1 0 0 'Failed option: 3'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 3'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 3'
+ LOG_ERROR 'Failed option: 3'
+ message='Failed option: 3'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 3'
Thu Feb  2 23:04:37 2023 - ERROR - Failed option: 3
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 107'
+ message='oe_test_nmon_04.sh line 107'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 107'
Thu Feb  2 23:04:38 2023 - ERROR - oe_test_nmon_04.sh line 107
+ return 0
+ expect
spawn nmon
4
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_4
+ CHECK_RESULT 1 0 0 'Failed option: 4'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 4'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 4'
+ LOG_ERROR 'Failed option: 4'
+ message='Failed option: 4'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 4'
Thu Feb  2 23:04:39 2023 - ERROR - Failed option: 4
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 118'
+ message='oe_test_nmon_04.sh line 118'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 118'
Thu Feb  2 23:04:40 2023 - ERROR - oe_test_nmon_04.sh line 118
+ return 0
+ expect
spawn sudo nmon
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "5\r""
+ test -f ./interactive_5
+ CHECK_RESULT 1 0 0 'Failed option: 5'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 5'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 5'
+ LOG_ERROR 'Failed option: 5'
+ message='Failed option: 5'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 5'
Thu Feb  2 23:04:41 2023 - ERROR - Failed option: 5
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 132'
+ message='oe_test_nmon_04.sh line 132'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 132'
Thu Feb  2 23:04:41 2023 - ERROR - oe_test_nmon_04.sh line 132
+ return 0
+ expect
spawn nmon
l6
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_6
+ CHECK_RESULT 1 0 0 'Failed option: 6'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 6'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 6'
+ LOG_ERROR 'Failed option: 6'
+ message='Failed option: 6'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 6'
Thu Feb  2 23:04:42 2023 - ERROR - Failed option: 6
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 143'
+ message='oe_test_nmon_04.sh line 143'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 143'
Thu Feb  2 23:04:43 2023 - ERROR - oe_test_nmon_04.sh line 143
+ return 0
+ expect
spawn nmon
l7
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_7
+ CHECK_RESULT 1 0 0 'Failed option: 7'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 7'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 7'
+ LOG_ERROR 'Failed option: 7'
+ message='Failed option: 7'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 7'
Thu Feb  2 23:04:44 2023 - ERROR - Failed option: 7
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 154'
+ message='oe_test_nmon_04.sh line 154'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 154'
Thu Feb  2 23:04:45 2023 - ERROR - oe_test_nmon_04.sh line 154
+ return 0
+ expect
spawn nmon
l8
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_8
+ CHECK_RESULT 1 0 0 'Failed option: 8'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 8'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 8'
+ LOG_ERROR 'Failed option: 8'
+ message='Failed option: 8'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 8'
Thu Feb  2 23:04:46 2023 - ERROR - Failed option: 8
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 165'
+ message='oe_test_nmon_04.sh line 165'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 165'
Thu Feb  2 23:04:46 2023 - ERROR - oe_test_nmon_04.sh line 165
+ return 0
+ expect
spawn nmon
l9
Error opening terminal: unknown.
send: spawn id exp3 not open
    while executing
"send "q""
+ test -f ./interactive_9
+ CHECK_RESULT 1 0 0 'Failed option: 9'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: 9'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: 9'
+ LOG_ERROR 'Failed option: 9'
+ message='Failed option: 9'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: 9'
Thu Feb  2 23:04:47 2023 - ERROR - Failed option: 9
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 176'
+ message='oe_test_nmon_04.sh line 176'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 176'
Thu Feb  2 23:04:48 2023 - ERROR - oe_test_nmon_04.sh line 176
+ return 0
+ expect
spawn nmon
 
qError opening terminal: unknown.
+ CHECK_RESULT 0 0 0 'Failed option: space'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Failed option: space'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ export NMON=cm
+ NMON=cm
+ expect
spawn nmon
Error opening terminal: unknown.
expect: spawn id exp3 not open
    while executing
"expect "Memory and Swap" {
        exec touch ./interactive_startUpControl_2
    }"
+ test -f ./interactive_startUpControl_1
+ CHECK_RESULT 1 0 0 'Failed option: start-up control'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Failed option: start-up control'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Failed option: start-up control'
+ LOG_ERROR 'Failed option: start-up control'
+ message='Failed option: start-up control'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed option: start-up control'
Thu Feb  2 23:04:49 2023 - ERROR - Failed option: start-up control
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmon_04.sh line 197'
+ message='oe_test_nmon_04.sh line 197'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmon_04.sh line 197'
Thu Feb  2 23:04:50 2023 - ERROR - oe_test_nmon_04.sh line 197
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Feb  2 23:04:51 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 12
+ test 12 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Feb  2 23:04:51 2023 - ERROR - failed to execute the case.
+ exit 12
+ post_test
+ LOG_INFO 'Start restore the test environment.'
+ message='Start restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start restore the test environment.'
Thu Feb  2 23:04:52 2023 - INFO  - Start restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpcvz23xsb ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpcvz23xsb
+ '[' 0 -ne 0 ']'
+ rm -rf './interactive_*'
+ unset NMON
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Feb  2 23:05:01 2023 - INFO  - End to restore the test environment.
